src/generators/operationGenerator.ts

```diff
@@ -150,11 +150,22 @@ function writeSpec(spec: OperationSpecDetails, writer: CodeBlockWriter): void {
   const urlParams = buildParameters(spec, "urlParameters");
   const headerParams = buildParameters(spec, "headerParameters");
   const formDataParams = buildParameters(spec, "formDataParameters");
-  const contentType = buildContentType(spec);
   const mediaType = buildMediaType(spec);
 
+  let targetMediaType: string | undefined = "";
```

Comment:
Comment from Jose: Add a comment to this line

Reply:
You can also initialize the variable to `undefined`, I do not think `""` is a valid media type.

Reply:
Done